docs/administrators/logging.rst

```diff
@@ -0,0 +1,145 @@
+.. _administrators_doc_logging:
+
+
+*******
+Logging
+*******
+
+This page describes the different logs files produced by the Inmanta server and its agents and explains what can be
+configured regarding to logging.
+
+
+Overview different log files
+============================
+
+By default log files are collected in the directory ``/var/log/inmanta/``. Three different types of log files exist: the
+server log, the resources action logs and the agent logs. The server log and the resource action log files are produced by
+the Inmanta server. The agent log files are produces by the Inmanta agents.
+
+
+Server log
+----------
+
+The ``server.log`` file contains general debugging information regarding the Imanta server. It shows information about actions
+performed by the Inmanta server (renewing parameters, purging resource action logs, etc.), API requests received by the
+Inmanta server, etc.
+
+
+Resource action logs
+--------------------
+
+The resource action log files contain information about actions performed on a specific resource. Each environment has one
+resource action log file. The filename of this log file looks as follows: ``<server
+.resource-action-log-prefix>-<environment-id>.log``. The prefix is configurable with the
+``purge-resource-action-logs-interval`` option as described in the :ref:`Inmanta configuration reference<config_reference>`.
```

Comment:
1. purge-interval to set prefix?
2. there is a built in command to directly refer to a config option

Reply:
Include "Model developer platform"